antsibull/data/docsite/list_of_collections_by_namespace.rst.j2

```diff
@@ -0,0 +1,22 @@
+:orphan:
+
+.. _list_of_collections_@{ namespace }@:
+
+{% set title = 'Collections in the ' ~ (namespace | title) ~ ' Namespace' | rst_ify -%}
+
+@{ title }@
+@{ '=' * title|length }@
+
+These are the collections with docs hosted on `docs.ansible.com <https://docs.ansible.com/>`_ in the **@{ namespace }@** namespace.
+
+{% for name in collections | sort %}
+* :ref:`@{ namespace }@.@{ name }@ <plugins_in_@{ namespace }@.@{ name }@>`
+{% endfor %}
+
+.. toctree::
+    :maxdepth: 1
+    :hidden:
+
+{% for name in collections | sort %}
+    @{ name }@/index
+{% endfor %}
```

Comment:
I'm not against this, but I do wonder what the usefulness is.  In other words, if we already have a list of collections which groups collections by namespace, what is the difference in having a page devoted to a list of just the colelctions inside of one of those namespaces?

Reply:
There are two reasons:
1. It makes sure that the intermediate URLs like https://docs.ansible.com/ansible/latest/collections/community/ return something that looks like part of the docsite, and not an Apache directory listing.
2. It makes sure that instead of having a full list of collections in the nav menu on https://ansible.fontein.de/collections/index.html, there's a list of collection namespaces.

If we'd use subsections in the index page for collection namespaces, we could achieve the same effect than 2 without this intermediate layer; I still prefer having these intermediate pages because of 1.

Reply:
I could see this being welcomed by partners who have multiple collections. But the main advantages to my mind are: reinforcing the hierarchy (a module belongs to a collection which belongs to a namespace) and avoiding directory listings.